README.md

```diff
@@ -8,7 +8,7 @@ Requirements
 
 Set zsh as your login shell:
 
-    chsh -s $(which zsh)
+    chsh --shell "$(which zsh)"
```

Comment:
That only works on GNU; we also have to support [macOS](https://www.unix.com/man-page/osx/1/chsh/) and other unix-like systems.